src/main/java/org/sonarsource/sonarlint/ls/settings/ServerConnectionSettings.java

```diff
@@ -38,7 +39,8 @@ public class ServerConnectionSettings {
   public static final String SONARCLOUD_URL = System.getProperty("sonarlint.internal.sonarcloud.url") != null ?
```

Comment:
SLCORE does not yet provide possibility to override US region URL. Once it does, we can be smarter about it too.